images/basespace/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.4
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
```

Comment:
Hail doesn't use the SHELL command, it inserts a script into the container, and runs it instead (so you could omit this).